src/main/java/org/apache/mesos/specification/TaskSet.java

```diff
@@ -0,0 +1,11 @@
+package org.apache.mesos.specification;
+
+import java.util.List;
+
+/**
+ * A TaskSet groups a set of Tasks that should be deployed using the same deployment strategy.
```

Comment:
The single-deployment-strategy aspect of a `TaskSet` just falls out of the fact that `PhaseSpecification` contains a `TaskSet`, right? That might be helpful to document so that developers can make that explicit connection.